<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateRequestWindow</class>
 <widget class="QWidget" name="CreateRequestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CreateRequestWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333;
color: #ffffff;</string>
  </property>
  <widget class="QLabel" name="label_equipment">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  Оборудование:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_typeOfFault">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип неисправности:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание проблемы:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>353</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>353</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_description">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_equipment">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_typeOfFault">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
